File: examples/factorial/factorial.cs
using System;

class Program
{
    static int Factorial(int n)
    {
        if (n < 2)
        {
            return n;
        }
        else
        {
            return n * Factorial(n - 1);
        }
    }
    
    static void Main()
    {
        int n = 7;
        int f = Factorial(n);
        Console.WriteLine(f);
    }
}

File: examples/factorial/factorial.java
public class Main
{
    static int factorial(int n)
    {
        if (n < 2)
        {
            return n;
        }
        else
        {
            return n * factorial(n - 1);
        }
    }
    
    public static void main(String[] args)
    {
        int n = 7;
        int f = factorial(n);
        System.out.println(f);
    }
}
